### DISCUSSION TITLE
2022-05-02 Gov Committee meeting

### DISCUSSION THREAD

**Discussion by mattjoyal** [cxndao/constellation#6] [Meeting Notes]
## Attendees

- MJ
- MG
- Matt Voska
- May

### Gov. Committee Agenda

- Committee member compensation
- Structure for proposals
- Goals

## Compensation

- One of the first proposals
- Defining job description & roles of the committees
    - Gov. committee can help with proposals & voting
    - Simplification of proposal process
        - Show how easy it is

### Proposal Structure

- Proposals - first one often the ground rules
    - Formatting, the process, how the voting process goes
    - Challenges - how are proposals written & made (templated) -
        - How is discussion facilitated? Chat or forum for discussion
            - GitHub for drafting, build a template
                - Source of truth
                - G Docs - broadest amount of participation
            - Then goes into Snapshot (offchain solution)
    - Example is Election Discord Channel & the ordering of it
- Discourse is used often (TS Connect backend = Discourse)
- Strict on use of threads & who can ping everyone
    - Gov. can only post, members can post in threads only
- Mgmt issue using multiple platforms - who is in charge of this?
    - GitHub - no email needed
    - Google - email needed

### Goals

- [ ]  this week - gather information, sketch out a couple solutions
    - Build GitHub Discussion on this
- [ ]  Figure out first proposals
- [ ]  Figure out voting process

### Continued Discussion

Reservations to using a process of GitHub?

- [ ]  new member walkthrough

- [ ]  task list?
- [[GitHub Issues](https://github.com/cxndao/constellation/issues)](https://github.com/cxndao/constellation/issues) section

- Challenge with DAOs
    - tough time attending all calls
    - Proposal about working, how to do things
    
- Why GitHub? Why G Docs?
    - what it’s useful for
    
Notion

- Check with Treasury/Programs to see what they are up to

Who sends the Town Hall Meeting calendar event?

- Who sends, what time?
- [ ]  Review date/time of Town Hall (conflict with TS events)

## Action Items

- [ ]  Review & draft ideas for goals in a GitHub Discussion Document
    1. Possible goal: processes & tools for proposal discussion, proposal drafting, & voting
    2. Possible goal: Complete committee compensation proposal
    3. Possible goal: Pass x number of proposals
- [x]  Review date/time of Town Hall as it conflicts with Techstars events

**Comment by mattjoyal** [comment]
Looks like GitHub does not have templates on Discussion currently.

**Reply by galligan** [reply]
Yeah, I noticed that. Bit of a bummer there, though maybe there's a workaround for it.
